package.json

```diff
@@ -9,11 +12,16 @@
   },
   "devDependencies": {
     "@wasm-tool/wasm-pack-plugin": "^0.4.2",
-    "copy-webpack-plugin": "^5.0.3",
-    "webpack": "^4.33.0",
+    "css-loader": "^3.2.1",
+    "file-loader": "^5.0.2",
+    "html-webpack-plugin": "^3.2.0",
+    "node-sass": "^4.13.0",
+    "rimraf": "^2.6.3",
+    "sass-loader": "^8.0.0",
+    "style-loader": "^1.0.1",
+    "webpack": "^4.41.2",
```

Comment:
For most of these I don't know what they are doing. We need to grab a cup of coffee together, so you can explain this to me. Until then, I'll just nod and merge.